### DISCUSSION TITLE
[Vote ended on 2023-03-21] Adopt a new upstream community website

### DISCUSSION THREAD

**Discussion by GregSutcliffe** [ansible-community/community-topics#208] [Votes done]
- Proposal & Discussion: #201 

Discussion should be kept to that issue, please.

This vote is to:
 
- Accept "Ansible Community" as a string describing the project at it's broadest level
- Register a DNS zone `ansible.community` for the community to use for all things relating to Ansible Community.
- Create a new website at the root of this zone to serve as a landing page for the Ansible community.
- Create a Website Working Group to oversee the creation and maintenance of the new website

Please vote with +1 to do this and -1 to keep things as they are (that is, keep using `ansible.com/community`).

**Comment by mariolenz** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by jillr** [comment]
+1

**Comment by chuckmilam** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by tremble** [comment]
+1

**Comment by rooso** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by ssbarnea** [comment]
-1 Im in favour of keeping `ansible.com/community`, or if there are real limitations using `community.ansible.com` subdomain instead.

Extra domains can cause lots of problems and confusions as anyone can register similar domains.

**Comment by AndreasDavour** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by apple4ever** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by pidreher** [comment]
-1

Prefer community.ansible.com

**Comment by markuman** [comment]
+1

**Comment by Landrash** [comment]
+1

**Comment by acozine** [comment]
+1 for registering the domain, but let's take the move slowly. See https://github.com/ansible-community/community-topics/issues/201#issuecomment-1470566420.

**Comment by tima** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by saito-hideki** [comment]
+1

**Comment by chofstede** [comment]
+1
